crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -174,8 +175,11 @@ pub struct CumulocityConverter {
     pub mqtt_schema: MqttSchema,
     pub entity_store: EntityStore,
     pub auth_proxy: ProxyUrlGenerator,
+    pub uploader_sender: LoggingSender<IdUploadRequest>,
     pub downloader_sender: LoggingSender<IdDownloadRequest>,
-    pub pending_operations: HashMap<CmdId, SmartRestOperationVariant>,
+    pub pending_upload_operations:
```

Comment:
That tuple is a bit too broad for people to easily make sense out of. I'd suggest a struct instead.